mapserv.h

```diff
@@ -41,6 +41,9 @@
 #include "maptile.h"
 
 #include "cgiutil.h"
+
+#include "cpl_conv.h"
```

Comment:
Not really a good practice to increase the number of includes in a global header, when itself doesn't require it. Should be the responsibility of each .c to include the needed headers

```suggestion
```